tests/animations/ripple.test.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Ripple from "libs/animations/Ripple";
+
+describe("animations/Ripple", () => {
+  it("can have children", () => {
+    const tree = renderer
+      .create(<div>Interact with me !</div>,{ ripple: true },)
```

Comment:
I dont understand how this is supposed to work 🤔

Reply:
https://zoapp.github.io/react-materialcomponents/index.html If you look at the github page , you can see a ripple at the top, it was written follow the rules that I've apply on thoses tests

Reply:
I do not understand how this code can work since, AFAIK, `renderer.create()` does not know what `ripple: true` means.